evalml/automl/automl_algorithm/default_algorithm.py

```diff
@@ -300,16 +273,9 @@ def _create_fast_final(self):
         pipelines = self._make_pipelines_helper(estimators)
 
         if self._split:
-            self._rename_pipeline_parameters_custom_hyperparameters(pipelines)
+            self._rename_pipeline_search_parameters(pipelines)
 
-        next_batch = []
-        for pipeline in pipelines:
-            parameters = self._create_select_parameters()
-            pipeline = pipeline.new(
-                parameters=self._transform_parameters(pipeline, parameters),
-                random_seed=self.random_seed,
-            )
-            next_batch.append(pipeline)
+        next_batch = self._create_n_pipelines(pipelines, 1)
 
         for pipeline in next_batch:
```

Comment:
We're creating the tuners twice here, once in `_create_n_pipelines` and then here in the `for pipeline in next_batch` for loop.

Reply:
Great catch!